tests/e2e/system_test/test_full_cluster_health.py

```diff
@@ -0,0 +1,286 @@
+
+        self.percent_to_fill = self.percent_to_fill - ceph_used_capacity_percent
+        logging.info(f"Percentage to fill is {self.percent_to_fill}%")
+
+        self.total_data_set = int(ceph_capacity * (int(self.percent_to_fill) / 100))
+        self.filesize = int(
+            self.crd_data["spec"]["workload"]["args"]["filesize"].replace("GiB", "")
+        )
+
+        # Make sure that filesize>=10 and servers<=60
+        self.servers = 60
+        self.filesize = int(self.total_data_set / self.servers)
+        if self.filesize < 10:
+            self.filesize = 10
+            self.servers = int(self.total_data_set / self.filesize)
+
+        self.crd_data["spec"]["workload"]["args"]["filesize"] = f"{self.filesize}GiB"
+        self.crd_data["spec"]["workload"]["args"][
+            "storagesize"
+        ] = f"{int(self.total_data_set)}Gi"
+        self.crd_data["spec"]["workload"]["args"]["servers"] = self.servers
+        self.crd_data["spec"]["workload"]["args"]["bs"] = "1024KiB"
+        self.crd_data["spec"]["workload"]["args"]["jobs"] = ["write", "read"]
+        self.crd_data["spec"]["workload"]["args"]["iodepth"] = 1
+
+    def cleanup(self):
+        """
+        Do cleanup in the benchmark-operator namespace.
+        delete the benchmark and make sure no PVC's and no PV's are left.
+
+        """
+        logger.info("Deleting FIO benchmark")
+        self.benchmark_obj.delete()
+        time.sleep(180)
+
+        # Getting all PVCs created in the test (if left).
+        NL = "\\n"  # NewLine character
+        command = ["oc", "get", "pvc", "-n"]
+        command.append(benchmark_operator.BMO_NAME)
+        command.append("-o")
+        command.append("template")
+        command.append("--template")
+        command.append("'{{range .items}}{{.metadata.name}}{{\"" + NL + "\"}}{{end}}'")
+        pvcs_list = run_command(command, out_format="list")
+        logger.info(f"list of all PVCs :{pvcs_list}")
+        for pvc in pvcs_list:
+            pvc = pvc.replace("'", "")
+            run_command(f"oc -n {benchmark_operator.BMO_NAME} delete pvc {pvc}")
+
+        # Getting all PVs created in the test (if left).
+        command[2] = "pv"
+        command[8] = (
+            "'{{range .items}}{{.metadata.name}} {{.spec.claimRef.namespace}}{{\""
+            + NL
+            + "\"}}{{end}}'"
+        )
+        command.remove("-n")
+        command.remove(benchmark_operator.BMO_NAME)
+        pvs_list = run_command(command, out_format="list")
+        logger.info(f"list of all PVs :{pvs_list}")
+
+        for line in pvs_list:
+            try:
+                pv, ns = line.split(" ")
+                pv = pv.replace("'", "")
+                if ns == benchmark_operator.BMO_NAME:
+                    logger.info(f"Going to delete {pv}")
+                    run_command(f"oc delete pv {pv}")
+            except Exception:
+                pass
+
+    def delete_pods(self):
```

Comment:
We have disruption_helpers in helpers for deleting these resource pods. Please check

Reply:
I'm doing the same - getting the required Pods and deletes them using Pod object